pdoc/extract.py

```diff
@@ -75,7 +83,7 @@ def walk_specs(specs: Sequence[Union[Path, str]]) -> dict[str, None]:
                 all_modules[m.name] = None
 
     if not all_modules:
-        raise ValueError(f"Module not found: {', '.join(str(x) for x in specs)}.")
+        raise ValueError(f"Module not found: {', '.join(str(x) for x in specs)}, or all modules ignored.")
```

Comment:
How about `f"No modules found matching spec: {', '.join(str(x) for x in specs)}"` ?

The common case here is to mistype a module name. "Module not found: typo, or all modules ignored" is a bit awkward.

Reply:
Fixed